scripts/sct_utils.py

```diff
@@ -391,6 +394,21 @@ def tmp_create(verbose=1):
     return path_tmp
 
 
+def delete_tmp_files_and_folders(path=''):
```

Comment:
would it be possible to add a brief comment to the function: what are the possible inputs? from the variable name it seems to be only path, but from the function name it seems like files could be input as well?